usecases/sharding/remote_index.go

```diff
@@ -285,7 +285,6 @@ func (ri *RemoteIndex) SearchAllReplicas(ctx context.Context,
 	cursor *filters.Cursor,
 	groupBy *searchparams.GroupBy,
 	adds additional.Properties,
-	replEnabled bool,
```

Comment:
this is unused, i assume we don't need to preserve the function signature of this public method for non-Weaviate users. i removed the replEnabled parameter in the Weaviate call.